toolchain/build-xenon-toolchain

```diff
@@ -376,94 +410,81 @@ function cube
 	exit 0	
 }
 
-function all_done
+all_done()
 {
 	RED='\e[0;31m'
 	NC='\e[0m'
 
 	echo
 	echo -e "All done, your xenon toolchain is located here : $PREFIX"
 	echo
-	echo -e "${RED}Please add the following path to your login script (~/.bashrc)"
+	echo -e "${RED}Please add the following path to your login script (~/.bashrc) if you'd like every shell to have access by default:"
 	echo
-	echo -e "export DEVKITXENON=\"$PREFIX\""
-	echo -e "export PATH=\"\$PATH:\$DEVKITXENON/bin:\$DEVKITXENON/usr/bin\""
+	echo -e "export DEVKITXENON=$PREFIX"
+	echo -e 'export PATH="${PATH:+${PATH}:}$DEVKITXENON/bin:$DEVKITXENON/usr/bin"'
+    echo 
+    echo -e "Alternatively, run "$0" env-cmd to install a command named xenon-env to /usr/local/bin/xenon-env to set those in the current shell for you"
 	echo -e "${NC}"
 }
 
-function check_build-essential
-{
-	echo -e "Ubuntu or Debian is detected."
-	dpkg -s build-essential >> $LOGFILE 2>&1
-
-	if [ $? -eq 1 ]; then
-		echo -e "The build-essential package was not detected on your system"
-		echo -e "To build the toolchain you need to download and install the build-essential package."
-		echo -e "Do you want this script to do it for you ? (y/n)"
-		read answer >> $LOGFILE 2>&1
-		if [ "$answer" == "y" ]; then
-			echo -e "Please wait while installing build-essential..."
-			apt install -y build-essential >> $LOGFILE 2>&1
-		fi
-	else
-		echo -e "The build-essential package was detected on your system"
-	fi
-
-	dpkg -s flex bison >> $LOGFILE 2>&1
-	if [ $? -eq 1 ]; then
-		echo -e "Flex / Bison were not detected on your system (binutils)"
-		echo -e "Do you want this script to attempt to install them for you? (y/n)"
-		read answer >> $LOGFILE 2>&1
-		if [ "$answer" == "y" ]; then
-			echo -e "Please wait while installing flex and bison..."
-			apt install -y flex bison >> $LOGFILE 2>&1
-		fi
-	fi;
-
-	dpkg -s gcc-multilib >> $LOGFILE 2>&1
-	if [ $? -eq 1 ]; then
-		echo -e "gcc-multilib was not detected on your system (binutils)"
-		echo -e "Do you want this script to attempt to install it for you? (y/n)"
-		read answer >> $LOGFILE 2>&1
-		if [ "$answer" == "y" ]; then
-			echo -e "Please wait while installing gcc-multilib..."
-			apt install -y gcc-multilib >> $LOGFILE 2>&1
-		fi
-	fi;
-}
-
 # start
-rm $LOGFILE &>/dev/null
+rm -f $LOGFILE
 
 if [ "$1" == "toolchain" ]; then
+    echo "Info: root privilages are required to install dependencies."
+
     if command -v apt-get &> /dev/null; then
-		check_build-essential
+		sudo apt install -y build-essential flex bison gcc-multilib
+    elif command -v dnf &> /dev/null; then
+        sudo dnf -y install flex bison glibc-devel.i686 libstdc++-devel.i686 gmp-devel mpfr-devel libmpc-devel texinfo git-core gcc g++ wget
     fi
-    toolchain_install
+    
+    toolchain_install "$0" "$@"
+    libxenon_install "$0" "$@"
```

Comment:
Changing this to also install libxenon is a breaking change with the docker images we create, there we have the toolchain separate from libxenon itself.

Due to this change that is no longer possible, it would be far nicer to leave them as they were, separated. The build script didn't at all mention that libxenon was installed at the same time as the toolchain, perhaps it was part of some old documentation somewhere?.

Please restore this part to install ONLY the toolchain as before.